include/networking/Common.h

```diff
@@ -2,6 +2,7 @@
 #define NETWORKING_COMMON_H
 
 #include <string>
+#include <vector>
```

Comment:
Just a test?

Reply:
This was MSVC (Microsoft's C++ compiler) complaining about includes